### ISSUE TITLE
Staging Review finding: Critical action link doesn't sufficiently indicate that it's a link

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#106784]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** AEDP
**Product:** Design Patterns
**Feature:** Critical Information and Service List Item Components

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In its default state, the critical action component consists of a yellow background, left-aligned bold black text, and a right-aligned chevron icon.

I don't believe that's sufficient to communicate that the element is a link. For a screen magnification user, the text of the link may be seen in isolation from the chevron icon --- meaning, it's just bold black text against a yellow background. That's especially true if the component extends to full width and the link text is relatively short, which the component seems to allow for.

Even if a user recognizes the component as clickable and not just plain text, the visual style of this link is remarkably similar to the visual style of a primary button: solid background color with a nearly-identical border radius (5px versus 4px), same font size and weight, and very similar left padding (16px versus 20px). I think a user might very reasonably assume this is a different style of button rather than a link, which breaks an interaction pattern for voice command software users ("Click button" not highlighting the element that they visually perceive as a button).

<details><summary>Link, screenshot or steps to recreate</summary>

![Left-most portion of a critical action component, with a single line of bold link text on a yellow background with rounded corners.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/105516-Brian-1.png?raw=true)

![Primary button, with a single line of bold text on a blue background with rounded corners.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/105516-Brian-2.png?raw=true)

</details>

## Recommended action

Include some kind of visual affordance that unambiguously communicates that this is a link --- like underlining it. Almost every other link inside the content of a VA.gov page is underlined, including the very similar [primary action link](https://dsva.slack.com/archives/C01DBGX4P45/p1742852235579389?thread_ts=1742485013.733889&cid=C01DBGX4P45), and I don't see what you would gain here from not underlining.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.4.1 Use of Colour - Level A](https://www.w3.org/TR/WCAG21/#use-of-color)
- **Modality:** Screen magnification
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bellepx0** [comment]
Fixed, added underline under link text. There is an open [PR](https://github.com/department-of-veterans-affairs/component-library/pull/1563) in component-library.

![Image](https://github.com/user-attachments/assets/581cb285-5018-41ea-879b-347b076fbf97)

**Comment by msbtterswrth** [comment]
confirmed in PR, ltgm.
